benchmarks/src/btreemap.rs

```diff
@@ -1,47 +1,252 @@
 use crate::count_instructions;
-use ic_stable_structures::{BTreeMap, DefaultMemoryImpl};
+use ic_cdk_macros::{query, update};
+use ic_stable_structures::{storable::Blob, BTreeMap, BoundedStorable, DefaultMemoryImpl};
 use tiny_rng::{Rand, Rng};
 
-/// Benchmarks inserting keys into a BTreeMap.
-#[ic_cdk_macros::query]
-pub fn btreemap_insert() -> u64 {
-    let mut btree: BTreeMap<u64, (), _> = BTreeMap::init(DefaultMemoryImpl::default());
+#[query]
+pub fn btreemap_insert_blob_4_1024() -> u64 {
```

Comment:
nit: We can probably reduce all this boilerplate with a simple macro that defines the functions for each combination.

Reply:
True. This code will evolve quite a bit in the next commits though, so I want to first wait until the code is more stable before working on removing the boilerplate.